base/tools/src/main/java/com/netscape/cmstools/nss/NSSCertFindCLI.java

```diff
@@ -29,15 +46,94 @@ public void printHelp() {
     }
 
     @Override
-    public void execute(CommandLine cmd) throws Exception {
+    public void createOptions() {
+        Option option = new Option(null, "cert", true, "Certificate to find");
+        option.setArgName("path");
+        options.addOption(option);
 
-        MainCLI mainCLI = (MainCLI) getRoot();
-        mainCLI.init();
+        option = new Option(null, "format", true, "Certificate format: PEM (default), DER");
+        option.setArgName("format");
+        options.addOption(option);
+    }
+
+    public Collection<X509Certificate> findCertByDERCert(byte[] derCert) throws Exception {
+
```

Comment:
Should never return more than one element so I would have preferred this to return an `X509Certificate`. The caller can add in the list.